xbmc/video/dialogs/GUIDialogAudioSubtitleSettings.cpp

```diff
@@ -156,6 +158,16 @@ void CGUIDialogAudioSubtitleSettings::OnSettingChanged(const CSetting *setting)
       g_application.m_pPlayer->SetAudioStream(m_audioStream);    // Set the audio stream to the one selected
     }
   }
+  else if (settingId == SETTING_VIDEO_STREAM)
```

Comment:
from each other. audio streams differ in language or codec. do video steams differ in quality or something?